artview/core/common.py

```diff
@@ -155,42 +155,49 @@ def startDisplay(self):
         return self.result, self.independent.isChecked()
 
 ########################
-# Table methods #
+#    Table methods     #
 ########################
 
 
 class CreateTable(QtGui.QTableWidget):
     """Creates a custom table widget."""
-    def __init__(self, column_names, name="Table",
+    def __init__(self, points, name="Table",
                  textcolor="black", bgcolor="gray", parent=None, *args):
```

Comment:
I like the idea behind it and it really cleans up a few parts of the code that bugged me (for example reading the 4th dimension of the data array to calculate the mean). I feel this is the way to go, but there may be potential tweaks down the road.